### ISSUE TITLE
PSR2 reporting the same issue twice/superfluous new ClosingBracePlacement sniff

### ISSUE THREAD

**Issue by jrfnl** [squizlabs/PHP_CodeSniffer#1929]
Commit https://github.com/squizlabs/PHP_CodeSniffer/commit/e1d3a5b097887e8f6eabf1e229bdbe17f9ba7dd8 added a new sniff called `PSR2.ControlStructures.ClosingBracePlacement` in response to issue #1857.

I'm not sure what purpose the new sniff serves. The PSR2 ruleset already includes the `Squiz.ControlStructures.ControlSignature` sniff which reports on the same issue.

AFAICS, the only problem was that that sniff did not correctly handle trailing comments after a brace.

Take the following example code:
```php
if ($something) {
}
else {
}

if ($something) {
}
// comment
else {
}

if ($something) {
} // comment
else {
}
```

This will now result in:
```
------------------------------------------------------------------------------------------
FOUND 5 ERRORS AFFECTING 5 LINES
------------------------------------------------------------------------------------------
  4 | ERROR | [x] Expected 1 space after closing brace; newline found
  5 | ERROR | [x] Control structure keyword must be on the same line as the closing brace
    |       |     from the earlier body
  9 | ERROR | [x] Expected 1 space after closing brace; newline found
 11 | ERROR | [ ] Control structure keyword must be on the same line as the closing brace
    |       |     from the earlier body
 16 | ERROR | [ ] Control structure keyword must be on the same line as the closing brace
    |       |     from the earlier body
```

In other words, all the messages are now thrown in duplicate, except for the test case with a trailing comment behind the closing brace of the `if`.
